src/OhDearWebhooksEventSubscriber.php

```diff
@@ -8,32 +8,30 @@
 class OhDearWebhooksEventSubscriber
 {
     public function onUptimeCheckFailed(
-        OhDearWebhookCall $ohDearWebhookCall,
-        OhDearUptimeStore $ohDearUptimeStore
+        OhDearWebhookCall $ohDearWebhookCall
```

Comment:
Why did you refactor away the injection of the store?

If this is not really needed, please revert these changes.

Reply:
Without that changes getting the mentioned fatal error, so that's why I refactored that. You can see details here: https://flareapp.io/share/Lm8E4VJP#F53